### ISSUE TITLE
"Add Package" button and "Package source" dropdown are appearing with green color border in High contrast black mode.

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/workbooks#363]
### Steps to Reproduce

1. Change the High contrast standards to High Contrast black before opening the application.
2. Launch Xamarin Workbooks application and create a new workbook.
3. Navigate to File -> Add Package. Invoke "Add Package" . It will open "Package Manager" window.
3. Type "Newton" in Search NuGet package edit box to get package list  and navigate to package list. Select any package to add.
4.Observe the border color of "Package Source" dropdown and "Add Package" button.
### Expected Behavior
Border should not be green color
### Actual Behavior
Add Package button and "Package source" dropdown are appearing with green color border in High contrast black mode.
Note: User may perceive them as disabled button.
### User Impact
As the border color appears in green color, user may perceive it as disabled element which is actually not. It deviates the user from the purpose.
### Screenshots
![15 hcb](https://user-images.githubusercontent.com/22956130/40724958-ea36d15c-643f-11e8-9bc9-21dfa66e1969.png)
![15 normal](https://user-images.githubusercontent.com/22956130/40724959-ea7989de-643f-11e8-8332-f0a6759b7276.png)

### Environment
Windows : Version 1803 (OS Build : 17134.81)
Xamarin Version : 1.4(master/b3f92f9)

**Comment by WiproPSAccTest** [comment]
Please add the following labels to the bug:
A11yMAS
Accessibility
Xamarin-Workbooks-Windows
Wipro
MAS4.3.1
